scripts/Muon/GUI/Common/plot_widget/base_pane/base_pane_view.py

```diff
@@ -0,0 +1,187 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from qtpy import QtWidgets
+
+from mantidqt.utils.qt import load_ui
+#from Muon.GUI.Common.plot_widget.plot_widget_view_interface import PlotWidgetViewInterface
+import Muon.GUI.Common.message_box as message_box
+
+ui_plotting_view, _ = load_ui(__file__, "base_pane_view.ui")
```

Comment:
I found out recently that the second return value here is the base widget of the ui form, so we can use this widget in the `class BasePaneView(base_widget, ui_plotting_view)`

Reply:
sorry, could you explain this to me a bit more?

Reply:
Sure, the second return value of `load_ui` is the `QWidget` found in the ui form. So we could do something like

```py
ui_plotting_view, widget = load_ui(__file__, "base_pane_view.ui")
class BasePaneView(widget, ui_plotting_view)
```